src/index.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './App';
+
+ReactDOM.render(
+  <React.StrictMode>
+    <App />
+  </React.StrictMode>,
+  document.getElementById('root'),
```

Comment:
- [x] Please make sure the index has 2 `<Link>`s in the navbar which leads to the booklist and categories locations.
The navbar might be in its own component, that's ok!


- [x] Please make sure the index has 2 `<Route>`s, one for the BookList component and the other for the Categories component. In order to not show a blank page when visiting the site for the first time, the BookList component should be the default page

 


I can see you have implemented this in App.j please move that here for now.

Reply:
@arslanbisharat I was not required to include the LInks or Routes in the index.js. Could you please enlighten me as to the purpose of moving them "for now"?